packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1650,7 +1650,7 @@ async function handler(
 
     // Use EventManager to conditionally use all needed integrations.
 
-    const updateManager = await eventManager.reschedule(evt, originalRescheduledBooking.uid, booking?.id);
```

Comment:
The third parameter `newBookingId` is only used when rescheduling seated bookings. This was causing seated logic on normal bookings